theme/themepacks.less

```diff
@@ -0,0 +1,62 @@
+/**************************************************************************
+    CSS theme variables
+    These are overridden by targets and used in: react-common, CRA webapps.
+    To add new theme variable:
+        1. Add variable to :root and .high-contrast below
+        2. Set the variable value in each target's theme/themepacks.less
+***************************************************************************/
+
+// TODO: Define the actual set of theme variables we want to support
+:root {
+    // Default to terrible choices so we can easily see when we're missing a theme variable in a target

```

Comment:
I'm fine with this for now, but I think long-term it'd be nice to have a "default theme" that actually looked nice, just to establish a baseline so that target themes don't have to override *everything* if they don't want to.

Reply:
Maybe a monochrome default.